src/main/resources/liquibase/042-updateDB.xml

```diff
@@ -21,4 +21,23 @@
     </changeSet>
     <!-- prefix for gen2 prostate pts in dev: 1771332079 on prod: -763171247  -->
     <!-- prod i.instance_id > 20000 are actual new added initial_biopsies for migrated pts  -->
-</databaseChangeLog>
\ No newline at end of file
+
+    <changeSet id="carrier_per_study_key" author="pegah">
+        <addColumn tableName="ddp_instance">
+            <column name="carrier_tracking_url" type="VARCHAR(500)"></column>
```

Comment:
This seems more like a `carrier_service` or `ddp_kit_request_settings` kind of thing to me.  Is `ddp_instance` the best place for this?

Reply:
moved to `carrier_service`